src/TestInfrastructure/Framework/TestLogger.cs

```diff
@@ -115,6 +115,11 @@ public void WriteLine(string messageFormat, params object[] args)
             WriteLine(string.Format(System.Globalization.CultureInfo.CurrentCulture, messageFormat, args));
         }
 
+        public void LogDebug(string message, params object[] args)
+        {
+            // no-op
```

Comment:
I think it's worth implementing it to dump to the console - It can be useful to see the output in the test to check that the output is being written correctly e.g.

```
            if (logToConsole)
            {
                Console.WriteLine("DEBUG: "+ ...);
            }
```

Reply:
Good suggestion, I'll merge as-is as I'm tired of the build failures and then open a new PR